srcStatic/ConsoleModuleLoader.cpp

```diff
@@ -18,57 +18,61 @@ ConsoleModuleLoader::ConsoleModuleLoader(const std::vector<std::string>& moduleN
 		return; // No console modules to load
 	}
 
-	// For now just handle the first name
-	if (moduleNames.size() > 1) {
-		throw std::runtime_error("ConsoleModuleLoader currently only supports a single loaded module");
+	// Temporary check. This will eventually become an error.
+	if (moduleNames.size() == 1 && moduleNames[0].empty()) {
+		return; // No console modules to load
 	}
-	auto moduleRelativeDirectory = moduleNames[0];
 
-	if (moduleRelativeDirectory.empty()) {
-		return; // No Console Module Loaded
+	if (std::any_of(moduleNames.begin(), moduleNames.end(), [](const std::string& moduleName){ return moduleName.empty(); })) {
+		PostError("All console module names must be non-empty.");
+		return;
 	}
 
-	auto moduleDirectory = fs::path(GetGameDirectory()).append(moduleRelativeDirectory).string();
-	moduleName = moduleRelativeDirectory;
+	for (auto& moduleName : moduleNames) {
```

Comment:
I would typically mark this const auto& to indicate no changes to container contents being looped through.